bot/exts/christmas/advent_of_code/_cog.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, bot: Bot) -> None:
         self.about_aoc_filepath = Path("./bot/resources/advent_of_code/about.json")
         self.cached_about_aoc = self._build_about_embed()
 
-        self.countdown_task = None
+        self.notification_task = None
         self.status_task = None
```

Comment:
We could just remove the of initialization of these names here, as we reassign them a few lines lower anyway. I think this is a leftover from a time where they may not have been assigned at `__init__` time.